index.html

```diff
@@ -3,11 +3,119 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="/styles.css" />
   </head>
   <body>
     <header>
       <h1>Quiz App</h1>
     </header>
+    <main>
+      <section class="content">
```

Comment:
You could probably add the styles of the `section` directly to your `main` element and get rid of the `section`.